knowTheDiff/aggregations.html

```diff
@@ -22,25 +22,29 @@
       <div class="wrapper" ng-controller="Controller">
         <div class="form-group first">
           <label class="col-md-4 control-label">Hashtag
-          </label>  
+          </label>
           <div class="col-md-6">
-            <input id="query" ng-model="query" type="text" placeholder="Search Loklak" class="form-control input-md query">
+            <input id="query" ng-model="query" type="text" placeholder="Search Loklak"
+                   class="form-control input-md query">
           </div>
-          <button href='#' class="btn btn-default" type="button" id="button" ng-click="SearchLoklak()">Search
+          <button href='#' class="btn btn-default" type="button" id="button"
+                  ng-click="SearchLoklak()">Search
           </button>
         </div>
         <div class="form-group second">
           <label class="col-md-4 control-label" for="from">Since Date
-          </label>  
+          </label>
           <div class="col-md-6">
-            <input id="since" name="since" type="date" ng-model="since" placeholder="Enter Since date here" class="form-control input-md fromDate"> 
+            <input id="since" name="since" type="date" ng-model="since"
+                   placeholder="Enter Since date here" class="form-control input-md fromDate"> 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/knowTheDiff/aggregations.html
+++ b/knowTheDiff/aggregations.html
@@ -36,7 +36,7 @@
           </label>
           <div class="col-md-6">
             <input id="since" name="since" type="date" ng-model="since"
-                   placeholder="Enter Since date here" class="form-control input-md fromDate"> 
+                   placeholder="Enter Since date here" class="form-control input-md fromDate">
           </div>
         </div>
         <div class="form-group third">
```